header.php

```diff
@@ -14,7 +14,7 @@
 	<?php tha_head_top(); ?>
 	<meta charset="<?php bloginfo( 'charset' ); ?>">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title><?php wp_title( '|', true, 'right' ); ?></title>
+	<title><?php bloginfo('name'); ?> | <?php is_front_page() ? bloginfo('description') : wp_title(''); ?></title>
 	<link rel="profile" href="http://gmpg.org/xfn/11">
 	<link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>">
 
```

Comment:
@nucro please click on the "details" links for the Travis reports to see how you can fix the errors in the PHP files.  It's usually very simple changes, such as adding a space or removing an extra line.

<img width="880" alt="screen shot 2015-12-01 at 12 08 27 pm" src="https://cloud.githubusercontent.com/assets/7597286/11507934/5ea36b62-9824-11e5-9406-731786b8bbb5.png">